docs/docs/cli.md

```diff
@@ -0,0 +1,63 @@
+# Goose CLI Commands
+
+Goose provides a command-line interface (CLI) with various commands to manage sessions, toolkits, and more. Below is a list of the available commands and their descriptions:
+
+## Goose CLI
+
+### `version`
+
+**Usage:**
+```sh
+  goose version
+```
+
+Lists the version of Goose and any associated plugins.
+
+### `session`
+
+#### `start`
+
+**Usage:**
+```sh
+  goose session start [--profile PROFILE] [--plan PLAN]
+```
+
+Starts a new Goose session.
+
+#### `resume`
+
+**Usage:**
+```sh
+  goose session resume [NAME] [--profile PROFILE]
```

Comment:
I don't remember if this exists, but if the profile doesn't have toolkits that were used in the session, there is potential for errors in the session. Maybe we put a warning here of this usage.